azure-pipelines.yml

```diff
@@ -0,0 +1,81 @@
+name: 3.0.0-ci.$(Date:yyyyMMdd).$(Rev:r) # equal to $(Build.BuildNumber)
+trigger:
+  branches:
+    include:
+    - dev
+    - master
+    - hotfix/*
+    - release/*
+variables:
+  DOTNET_CLI_TELEMETRY_OPTOUT: 1
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+  DOTNET_SDK_VERSION: '3.0.100-preview3-010431'
+jobs:
+- job: Linux
+  pool:
+    vmImage: ubuntu-16.04
```

Comment:
It's pretty disappointing they're still stuck on 16.04... FWIW here's the issue: https://github.com/Microsoft/azure-pipelines-image-generation/issues/506, go vote :)

(but it doesn't seem blocking in any way either)